libs/designsystem/src/lib/components/page-local-navigation/page-local-navigation-item.ts

```diff
@@ -0,0 +1,26 @@
+import { ThemeColor } from '@kirbydesign/core';
+
+export type LocalNavigationItemBadge = {
+  icon?: string;
+  content?: string;
+  description?: string;
+  isCustomIcon?: boolean;
+  themeColor: ThemeColor;
+};
+
+export interface LocalNavigationItem {
+  /**
+   * Text of the item
+   */
+  text: string;
+
+  /**
+   * Id of the item. Must be unique!
+   */
+  id: string;
```

Comment:
This component doesn't seem to require a unique id - remove and let the client decide it if it needs one.